compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPULowerExecutableTarget.cpp

```diff
@@ -206,6 +206,10 @@ void LLVMCPULowerExecutableTargetPass::runOnOperation() {
               CPUDoubleTilingExpert:
             addDoubleTilingExpertPassPipeline(nestedModulePM, lowerToAVX2);
             break;
+          case IREE::Codegen::DispatchLoweringPassPipeline::
+              CPUMatmulDoubleTilingExpert:
+            addMatmulDoubleTilingExpertPassPipeline(nestedModulePM);
```

Comment:
In addition to the flag comment that Mahesh mentioned, probably good to make this function name have "Padding" in it to differentiate it from the default one and peeling, if we plan to keep separate pipelines.

Reply:
+1